modules/platform/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/operations/document/CopyDocumentTest.java

```diff
@@ -129,4 +130,27 @@ public void testLifeCycleResetOnCopy() throws OperationException {
             assertEquals(initialLifeCycleState, result.getCurrentLifeCycleState());
         }
     }
+
+    @Test
+    public void testVersionNotNullOnCopy() throws OperationException {
+        DocumentModel source = session.createDocumentModel(ROOT, "Source", FILE);
+        source = session.createDocument(source);
+        source = session.getDocument(session.checkIn(source.getRef(), VersioningOption.MAJOR, ""));
+
+        assertEquals(1, (long) source.getPropertyValue("uid:major_version"));
+        assertEquals(0, (long) source.getPropertyValue("uid:minor_version"));
```

Comment:
In fact, is there any point in versioning the created document and verifying its version numbers?
Aren't e only interested in the copied document in this test?